cpp/src/com/google/idea/blaze/cpp/BlazeCWorkspace.java

```diff
@@ -126,6 +127,7 @@ public void update(
     BlazeConfigurationResolverResult newResult =
         configurationResolver.update(
             context, workspaceRoot, projectViewSet, blazeProjectData, oldResult);
+    BlazeCompilerInfoMap.getInstance(project).setState(newResult.targetToCompilerVersion);
```

Comment:
Not a review comment but just for me to understand, will this map be populated on Startup syncs? If the IDE is closed then reopened the plugin picks up the last sync state from disk, will this map be loaded from there as well or it will require manual sync?

Reply:
I see, thank you.